# PicoHand.msg
# 这个消息用于传输PICO手部的所有关键点

Header header
geometry_msgs/Point[] keypoints